src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/SVVCFWriter.java

```diff
@@ -43,13 +46,21 @@ public static void writeVCF(final List<VariantContext> localVariants, final Stri
         writeVariants(vcfFileName, sortedVariantsList, referenceSequenceDictionary);
     }
 
-    private static void logNumOfVarByTypes(final List<VariantContext> sortedVariantsList, final Logger logger) {
+    private static void logNumOfVarByTypes(final List<VariantContext> variants, final Logger logger) {
 
-        logger.info("Discovered " + sortedVariantsList.size() + " variants.");
+        logger.info("Discovered " + variants.size() + " variants.");
 
-        sortedVariantsList.stream()
-                .collect(Collectors.groupingBy(vc -> (String)vc.getAttribute(GATKSVVCFConstants.SVTYPE), Collectors.counting()))
-                .forEach((key, value) -> logger.info(key + ": " + value));
+        final Map<String, Long> variantsCountByType = variants.stream()
+                .collect(Collectors.groupingBy(vc -> (String) vc.getAttribute(GATKSVVCFConstants.SVTYPE), Collectors.counting()));
+
+        variantsCountByType.forEach((key, value) -> logger.info(key + ": " + value));
+
+        final Set<String> knownTypes = new HashSet<>( EnumUtils.getEnumMap(SimpleSVType.TYPES.class).keySet() );
+        knownTypes.add("BND_INV33");
```

Comment:
Are the any constants for this? If not create them please.

Reply:
done